### PR TITLE
Sdh okta210931 postman

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Updated this guide
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-210931](https://oktainc.atlassian.net/browse/OKTA-210931)

### CODE REVIEW

File: packages/@okta/vuepress-site/code/rest/index.md [hunk 1]
```diff
@@ -1,94 +1,100 @@
...
 integration: back-end
 icon: code-rest
 meta:
   - name: description
-    content: Find documentation to get started with Okta REST APIs and learn how to import a collection and send requests in Postman.
+    content: Get started with Okta REST APIs and learn how to import a collection and send requests in Postman.
 ---
 
-A great way to learn an API is to issue requests and inspect the responses. You can easily use our Postman collections to do just that.
+A great way to learn an API is to issue requests and inspect the responses. You can easily use our Postman collections to do just that. To use these collections, you need to set up your local environment and import a collection. Then, you can send a test request and verify the results.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/code/rest/index.md]: "Then you can send ..." - no comma needed here.

File: packages/@okta/vuepress-site/code/rest/index.md [hunk 2]
```diff
@@ -1,94 +1,100 @@
...
+> **Note:** Alternatively, you can [create an OAuth 2.0 access token](/docs/guides/implement-oauth-for-okta/) for use with a number of Okta endpoints. This is a feature that allows you to interact with Okta APIs using scoped OAuth 2.0 access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
 
-1. [Install the Postman app](https://www.getpostman.com/apps).
-1. Launch Postman and click the **Import** button. Select **Import From Link**, and paste this link into the textbox: `https://developer.okta.com/docs/api/postman/example.oktapreview.com.environment`
-![Importing the Okta Example Environment](/img/import_enviro.png "Importing the Okta Example Environment")
+2. [Install the Postman app](https://www.getpostman.com/apps).
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/code/rest/index.md]: If you use just "1." as the marker, you won't have to renumber the steps if something changes in the future. Markdown will automatically number them in sequential order.

File: packages/@okta/vuepress-site/code/rest/index.md [hunk 3]
```diff
@@ -1,94 +1,100 @@
...
-1. Select the **Collections** tab in Postman and open the **Users (Okta API)** collection. Open the **List Users** folder, and select **(GET) List Users**. This loads the List Users request into Postman, ready to send.
+1. Select the **Collections** tab in Postman and expand the **Users (Okta API)** collection.
+1. Expand the **List Users** folder and select **List Users**. This loads the List Users request into Postman, ready to send.
 1. Click **Send**. The result pane automatically displays the results of your request:
-![GET List Users](/img/postman_response.png "GET List Users")
 
 If you receive an error, it's likely that one of the values in the environment isn't set correctly. Check the values and try again.
 
-Once you have completed this simple request, you're ready to explore the Okta API!
+Once you have completed this simple request, you're ready to explore the Okta API.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/code/rest/index.md]: "After you have ...

File: packages/@okta/vuepress-site/code/rest/index.md [hunk 4]
```diff
@@ -1,94 +1,100 @@
...
 
-Your imported collections contain URLs and JSON request bodies that have sample data with variables such as `{userId}`.
-Replace URL and body variables with the IDs of the resources you wish to specify.
+Your imported collections contain URLs and JSON request bodies that have sample data with variables such as `{userId}`. You can replace URL and body variables with the IDs of the resources that you want to specify.
 
 To find an ID:
 
-1. List a resource collection with a search or filter. For example, list the users in your org, as you did in the previous section.
-![List Users Response](/img/postman_response2.png "List Users Response")
+1. To get a user's ID, for example, send a request to list the users in your org like you did in the previous section. Each user listed in the response has an ID:
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/code/rest/index.md]: I think we need to keep that first sentence ("List a resource collection with a search or filter.") to retain the general idea of what is being done in the specific example.

File: packages/@okta/vuepress-site/code/rest/index.md [hunk 5]
```diff
@@ -1,94 +1,100 @@
...
+
+![List Users Response](/img/postman_response2.png "Response example for a GET users request that highlights the ID in the response")
 
-1. Copy the `id` of the resource, in this example the `id` for Tony Stark, in your next request.
+2. Copy the `id` of the resource, in this example the `id` for Tony Stark, for use in your next request.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/code/rest/index.md]: Keep the "1." as the markers to make your future life easier.

File: packages/@okta/vuepress-site/code/rest/index.md (hunk 2 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/code/rest/index.md]: @IanHakes-okta that doesn't seem to be working right now, so I am numbering. If you look at the current guide - the numbering is broken: https://developer.okta.com/code/rest/

File: packages/@okta/vuepress-site/code/rest/index.md (hunk 3 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/code/rest/index.md]: gah missed that one!

**Pull request merged**